assets/js/translator.js

```diff
@@ -125,7 +127,7 @@ if(modeEnabled('translation')) {
 
             timer = setTimeout(function () {
                 if($('#instantTranslation').prop('checked')) {
-                    translateText();
+                    translate();
```

Comment:
Um, nope. `translate()` won't fire it as the condition checks for being present on that interface. This is required only in the event that if the user types in a `url`, I take him to that page. I believe this would not cause problems.